Gruntfile.js

```diff
@@ -1,41 +1,77 @@
+/* global module */
```

Comment:
hey @spiray - question - is the tab space set to 4 on your editor or is it the lint settings?

Reply:
Setting it on eslint is perfect. I think if we make it `"indent": [2, 2]` then, it will be 2 spaces all over.